src/go/rpk/pkg/cli/cmd/group/describe.go

```diff
@@ -64,9 +66,11 @@ information about the members.
 				described,
 				fetched,
 				listed,
+				summary,
 			)
 		},
 	}
+	cmd.Flags().BoolVarP(&summary, "print-summary", "s", false, "Print the group summary section")
```

Comment:
`Print only the group summary section`? Since by default, we print the summary and the detailed per partition part.